pyrate/core/timeseries.py

```diff
@@ -302,6 +303,110 @@ def _solve_ts_lap(nvelpar, velflag, ifgv, mat_b, smorder, smfactor, sel, vcmt):
     return tsvel
 
 
+def linear_rate_pixel(y, t):
+    """
+    Calculate best fitting linear rate to cumulative displacement
+    time series for one pixel using linear regression.
+
+    :param ndarray y: 1-dimensional vector of cumulative displacement time series
+    :param ndarray t: 1-dimensional vector of cumulative time at each epoch
+
+    :return: linrate: Linear rate; gradient of fitted line.
+    :rtype: float
+    :return intercept: Y-axis intercept of fitted line at t = 0
+    :rtype: float
+    :return: rsquared: R-squared value of the linear regression
+    :rtype: float
+    :return: error: Standard error of the linear regression
+    :rtype: float
+    :return: samples: Number of observations used in linear regression
+    :rtype: int
+    """
+
+    while True:
+        try:
```

Comment:
don't need the `while True` and `break`? just `try and except` are enough?

Reply:
Removed. Thanks for the tip! I followed a recipe that had them in